packages/common/infra/src/framework/core/identifier.ts

```diff
@@ -100,7 +99,7 @@ export function createIdentifier<T>(
 export function createIdentifierFromConstructor<T extends Component>(
   target: Type<T>
 ): Identifier<T> {
-  return createIdentifier<T>(`${target.name}${stableHash(target)}`);
+  return createIdentifier<T>(target.name);
```

Comment:
it work, but not safe

Reply:
why

Reply:
If there are types with the same name, this can overwriting.

Reply:
then use type itself as identifier

Reply:
Then hmr is broken, once you use the identifier bound to the type

Reply:
<div class='graphite__hidden'>
          <div>🎥 Video uploaded on Graphite:</div>
            <a href="https://app.graphite.dev/media/video/g3jz87HxbjOJpXV3FPT7/40fa720d-d44e-4396-b4a0-ea89a89be683.mp4">
              <img src="https://app.graphite.dev/api/v1/graphite/video/thumbnail/g3jz87HxbjOJpXV3FPT7/40fa720d-d44e-4396-b4a0-ea89a89be683.mp4">
            </a>
          </div>
<video src="https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/g3jz87HxbjOJpXV3FPT7/40fa720d-d44e-4396-b4a0-ea89a89be683.mp4">CleanShot 2024-09-04 at 15.20.13.mp4</video>